wily/commands/rank.py

```diff
@@ -0,0 +1,128 @@
+"""
+Rank command.
+
+The report command gives a table of files sorted according their ranking scheme
+of a specified metric.
+Will compare the values between files and return a sorted table.
+
+
+TODO: Refactor individual file work into a separate function
+TODO: Layer on Click invocation in operators section, __main__.py file
+TODO: Is the a better way to retrieve the revision number than via index?
+"""
+import tabulate
+import os
+import operator as op
+
+from wily import logger, format_date, format_revision, MAX_MESSAGE_WIDTH
+from wily.config import DEFAULT_GRID_STYLE
+from wily.operators import resolve_metric, MetricType
+from wily.state import State
+
+
+def rank(config, path, metric="maintainability.index", revision_number=0):
```

Comment:
revision keys are strings (the git-sha), in this case you're referring to the index. So call it revision_index

Reply:
renamed the parameter to revision_index in the commit that was just pushed